src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
```

Comment:
We can't use the spread operator with "use strict" ...the internet told us...so it obviously must be true. That was what we came up with after tracking down our syntax errors that started showing up.